dependency_support/load_external.bzl

```diff
@@ -38,60 +38,6 @@ def load_external_repositories():
     repo_llvm()
     repo_rules_hdl()
 
-    # Release 2024-01-22, current as of 2024-06-26
-    # zlib is added automatically by gRPC, but the zlib BUILD file used by gRPC
-    # does not include all the source code (e.g., gzread is missing) which
-    # breaks other users of zlib like iverilog. So add zlib explicitly here with
-    # a working BUILD file.
-    # Needs to be early in this file to make sure this is the version
-    # picked -- Version 1.3.x fixes function prototype warnings in c++20.
-    http_archive(
-        name = "zlib",
-        sha256 = "50b24b47bf19e1f35d2a21ff36d2a366638cdf958219a66f30ce0861201760e6",
-        strip_prefix = "zlib-1.3.1",
-        urls = [
-            "https://github.com/madler/zlib/archive/v1.3.1.zip",
-        ],
-        build_file = "//dependency_support/zlib:bundled.BUILD.bazel",
-    )
-
-    # V 1.14.0 (released 2023-08-02, current as of 2024-06-26)
-    http_archive(
-        name = "com_google_googletest",
-        urls = ["https://github.com/google/googletest/archive/refs/tags/v1.14.0.zip"],
-        strip_prefix = "googletest-1.14.0",
-        sha256 = "1f357c27ca988c3f7c6b4bf68a9395005ac6761f034046e9dde0896e3aba00e4",
-    )
-
-    # LTS 20240116.2 (released 2024-04-08, current as of 2024-06-26)
-    http_archive(
-        name = "com_google_absl",
-        urls = ["https://github.com/abseil/abseil-cpp/archive/refs/tags/20240116.2.tar.gz"],
-        strip_prefix = "abseil-cpp-20240116.2",
-        sha256 = "733726b8c3a6d39a4120d7e45ea8b41a434cdacde401cba500f14236c49b39dc",
-    )
-
-    # Released 2024-06-03, current as of 2024-06-26
-    # Protobuf depends on Skylib
-    # Load bazel skylib as per
-    # https://github.com/bazelbuild/bazel-skylib/releases
-    http_archive(
-        name = "bazel_skylib",
-        urls = [
-            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.7.1/bazel-skylib-1.7.1.tar.gz",
-        ],
-        sha256 = "bc283cdfcd526a52c3201279cda4bc298652efa898b10b4db0837dc51652756f",
-    )
-
-    http_archive(
-        name = "boringssl",
-        # Commit date: 2024-06-24
-        # Note for updating: we need to use a commit from the main-with-bazel branch.
-        strip_prefix = "boringssl-e6b03733628149a89a1d18b3ef8f39aa1055aba8",
-        sha256 = "006596f84d9cc142d9d6c48600cf6208f9d24426943b05e8bcda06e523f69dc8",
-        urls = ["https://github.com/google/boringssl/archive/e6b03733628149a89a1d18b3ef8f39aa1055aba8.tar.gz"],
-    )
-
     # Commit on 2023-02-09
     http_archive(
```

Comment:
do we still need that? (I thought we removed the python bindings but maybe one of our dependency uses it?)

Reply:
Maybe not. Since I didn't know what it is for, I didn't touch it.

Reply:
I'm pretty sure everything using pybind is gone.